contracts/disputes/DisputeManager.sol

```diff
@@ -628,39 +666,42 @@ contract DisputeManager is DisputeManagerV1Storage, GraphUpgradeable, IDisputeMa
      * @param _disputeID ID of the dispute to resolve
      * @return Dispute
      */
-    function _resolveDispute(bytes32 _disputeID) private returns (Dispute memory) {
-        require(isDisputeCreated(_disputeID), "Dispute does not exist");
+    // function _resolveDispute(bytes32 _disputeID) private returns (Dispute memory) {
+    //     require(isDisputeCreated(_disputeID), "Dispute does not exist");
 
-        Dispute memory dispute = disputes[_disputeID];
+    //     Dispute memory dispute = disputes[_disputeID];
 
-        // Resolve dispute
-        delete disputes[_disputeID]; // Re-entrancy
+    //     // Resolve dispute
+    //     delete disputes[_disputeID]; // Re-entrancy
 
-        return dispute;
-    }
+    //     return dispute;
+    // }
 
     /**
      * @dev Returns whether the dispute is for a conflicting attestation or not.
      * @param _dispute Dispute
      * @return True conflicting attestation dispute
      */
-    function _isDisputeInConflict(Dispute memory _dispute) private pure returns (bool) {
-        return _dispute.relatedDisputeID != 0;
+    function _isDisputeInConflict(Dispute memory _dispute) private view returns (bool) {
+        bytes32 relatedID = _dispute.relatedDisputeID;
+        return
+            relatedID != 0 && disputes[relatedID].status == IDisputeManager.DisputeStatus.Pending;
```

Comment:
Not sure why this was added? I think as long as the relatedDisputeID is set it should be sufficient to know there's a conflicting dispute?

Reply:
Ah, so now I see this is for the `_isDisputeInConflict` check to return false when rejecting the related dispute. This makes sense, but it would be good to add a comment here to explain it.